example/config1.yaml

```diff
@@ -0,0 +1,198 @@
+logging:
```

Comment:
looks like the three files are identical except for the hostID section. Any chance of reusing them somehow (e.g. provide a 1 file go utility to generate the log based on input)?

Reply:
Sounds good- will address this in a subsequent pr.